libtest/mock-transport.c

```diff
@@ -124,6 +146,22 @@ log_transport_mock_read_method(LogTransport *s, gpointer buf, gsize count, LogTr
   return count;
 }
 
+static gssize
+log_transport_mock_write_method(LogTransport *s, const gpointer buf, gsize count)
+{
+  LogTransportMock *self = (LogTransportMock *)s;
+  g_array_append_vals(self->write_buffer, buf, count);
+  return count;
+}
+
+static void
+log_transport_mock_free_method(LogTransport *s)
+{
+  LogTransportMock *self = (LogTransportMock *)s;
+  g_array_free(self->write_buffer, TRUE);
+  log_transport_free_method(s);
```

Comment:
In `mock_init()`, `log_transport_init_instance` can be called (holds symmetry with `log_transport_free_method`).

Reply:
Done thanks.